lib/mllp/client.ex

```diff
@@ -338,6 +342,7 @@ defmodule MLLP.Client do
       |> validate_options()
       |> maybe_set_default_options()
       |> put_socket_address()
+      |> Map.put(:mllp_trailer, :binary.compile_pattern(@trailer))
```

Comment:
Compile a binary pattern using the MLLP trailer and use this for fragmented response validation since we'll do this indefinitely.